playbooks/02 - Use Case - SOC Simulator/Run Selected Scenario.json

```diff
@@ -4,7 +4,7 @@
     "name": "Run Selected Scenario",
     "aliasName": null,
     "tag": null,
-    "description": null,
+    "description": "Execute a scenario and create its related records",
```

Comment:
Execute a scenario and create its related records --> Execute scenario to create respective records? OR Execute a scenario and create related records

Reply:
@cs-suyog-jinturkar Updated PB description to "Execute a scenario and create related records"